aggregator/src/binary_utils/job_driver.rs

```diff
@@ -158,14 +159,44 @@ where
             // (we'll pick up any additional jobs on the next iteration of this loop). We can't
             // overestimate since this task is the only place that permits are acquired.
```

Comment:
nit: I think this should say "lease" instead of "permit". While you're in here, could you check whether we're being consistent about that term in this file?

Reply:
I just changed it here. It's a bit tricky--in this context most occurrences of "permit" are referring to acquisition of SemaphorePermits.